scheduler/feasible.go

```diff
@@ -448,8 +443,9 @@ func resolveTarget(target string, node *structs.Node) (interface{}, bool) {
 	}
 }
 
-// checkConstraint checks if a constraint is satisfied
-func checkConstraint(ctx Context, operand string, lVal, rVal interface{}) bool {
+// checkConstraint checks if a constraint is satisfied. The lVal and rVal
```

Comment:
Need to do the same here too: https://github.com/hashicorp/nomad/blob/136dace45c34223bb0a63ba7a0019792d6e7d797/scheduler/feasible.go#L984